demo/midgard-node/src/commands/listen.ts

```diff
@@ -378,6 +416,7 @@ const router = HttpRouter.empty.pipe(
   HttpRouter.get("/commit", getCommitEndpoint),
   HttpRouter.get("/merge", getMergeHandler),
   HttpRouter.get("/reset", getResetHandler),
+  HttpRouter.get("/cbors", getCBORsWithAddressHandler),
```

Comment:
As discussed in chat:
```suggestion
  HttpRouter.get("/txs", getTxsOfAddressHandler),
```